src/sfizz/Voice.cpp

```diff
@@ -661,6 +661,8 @@ void sfz::Voice::fillWithGenerator(AudioSpan<float> buffer) noexcept
     } else {
         const auto numFrames = buffer.getNumFrames();
 
+        buffer.fill(0.0f);
```

Comment:
I think it's filled at the beginning of renderBlock?

Reply:
Voices are rendered into a temporary buffer. This could go, however this leads to another problem:
Synth.cpp `tempSpan->fill(0.0f);` -> called once per cycle
The `tempSpan` will have to be cleared each time before the voice rendering.